client/src/pages/Login/Login.tsx

```diff
@@ -39,10 +40,12 @@ export default function Login(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid item xs={12} sm={8} md={6} elevation={0} component={Paper}>
+        <img src={BG_img} style={{ height: '100%' }} />
```

Comment:
Bit of a nit pick here but `bgImg` instead of `BG_img` 

Snake_case is not JavaScript convention, camelCase is.